nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -243,6 +246,75 @@ export function EditSubscriberButton(props: EditProps) {
   );
 }
 
+export function BulkDeleteSubscriberButton(props: {onClose: () => void}) {
+  const classes = useStyles();
+  const successCountRef = useRef(0);
+  const enqueueSnackbar = useEnqueueSnackbar();
+  const ctx = useContext(SubscriberContext);
+
+  const fileInput = useRef(null);
+  const deleteSubscribers = async subscribers => {
+    successCountRef.current = 0;
+    for (const subscriber of subscribers) {
+      try {
+        await ctx.setState?.(subscriber.imsi);
+        successCountRef.current = successCountRef.current + 1;
+      } catch (e) {
+        const errMsg = e.response?.data?.message ?? e.message ?? e;
+        enqueueSnackbar('error deleting ' + subscriber.imsi + ' : ' + errMsg, {
+          variant: 'error',
+        });
+        return;
+      }
+    }
+    enqueueSnackbar(
+      ` Subscriber${
+        successCountRef.current > 1 ? 's ' : ''
+      } deleted successfully`,
```

Comment:
```suggestion
      'Subscriber(s) deleted successfully',
```

nit: This is just an opinion of mine, but I don't think the extra complexity is worth it here for things like this.